app/models/project.rb

```diff
@@ -64,8 +64,6 @@ def satellitedir
 
   # Push the existing contents of the satellite repo to the bare repo
   def pushtobare
-    barerepo = Rugged::Repository.new self.barerepopath
-    satelliterepo = Rugged::Repository.new self.satelliterepopath
```

Comment:
I believe those 2 lines have no use, there are functions defined called barerepo and satelliterepo that return the same thing